README.md

```diff
@@ -30,7 +31,9 @@ console.log(password);
 
 #### `generateMultiple(amount, options)`
```

Comment:
Do you think you could change this to how it's done in the [node docs](https://nodejs.org/api/http.html#http_request_write_chunk_encoding_callback), like this?
```markdown
#### generateMultiple(amount[, options])`
```
Then we can do away with the "If the options parameter is not provided, yadda yadda.." which I think is somewhat redundant. Please do that with the above one too please :)